New add-on with price per day per person

nicolocaruso85 opened the pull request:
A roomify manager should be able to define an add-on for Types with a price/night/person. The perfect exemple is the tourism tax that you can found in France (maybe in some other countries).

<img width="880" alt="capture d ecran 2017-04-12 a 10 26 06" src="https://cloud.githubusercontent.com/assets/23718178/24948445/a1c5b214-1f6a-11e7-9eb9-d02059292d49.png">

You will find attached an experimental implementation of this new add-on.
[Archive.zip](https://github.com/Roomify/RfA/files/915893/Archive.zip)

flodaq commented:
Also, we should be able to define a numeric price value instead of a strict integer.

Commit:
New add-on with price per person

Commit:
Fix tests

salvoscala committed:
Improve guests filter on the availability search. (#184)

* Update the variable with the number of the biggest type on types update

* Fix typo

Commit:
Fix “Metatag page title on 'listing/%type_id' is empty or wrong” (#192)

Commit:
Improve admin theme to don’t hide metatags fieldsets (#191)

salvoscala committed:
Sidebar menu and documentation icon disappear when visiting the site not in english. (#178)

* Info icon on different language pages.

* Language fallback for the dashboard menu links

* Fix tests (#181)

* Improve controls

* Send a notification email to Roomify managers for Contact form submissions (#180)

* Send a notification email to Roomify managers for Contact form submission

* Fix tests (#181)

* Export rule

* Code style.

* Create link to enquiries and improve theme.

* Add issue template

* Guest Reviews (#157)

* Add the Guest Reviews feature.

* Integrate average rating.

* Improve Reviews

* Test Reviews

* Email guest after checkout

* Notify the property owner when a new review has been submitted.

* Add Owner Reviews view

* Add reviews on single property page.

* Integrate reviews to base themes.

* Property Owner should be able to access the reviews.

* Owner should be also able to publish the review.

* Add the Guest Reviews feature.

* Integrate average rating.

* Improve Reviews

* Rebase fixes

* Add reviews in the dashboard.

* Code style.

* Hide the properties reference field when a property is set.

* Allow property owner to publish reviews.

* Fix tests

* Minor changes.

* Fix Contact Submissions menu link position.

* Export locanda/casa listing panel and fix code style.

* Check permission when show 'Create booking' button (#190)

* Fix redirect on the “Quick Setup” page when try to delete example content (#187)

* Use the default currency for new enquiries (#173)

* Fix update of pricing rates when default currency has no decimals (#183)

* Fix tests

* Improve guests filter on the availability search. (#184)

* Update the variable with the number of the biggest type on types update

* Fix typo

* Fix “Metatag page title on 'listing/%type_id' is empty or wrong” (#192)

* Improve admin theme to don’t hide metatags fieldsets (#191)

Commit:
Downgrade i18n to 1.15 (#194)

Commit:
Always create a bat type when creating a Property. (#179)

* Always create a BAT Type when creating a Property

* Downgrade i18n to 1.15 (#194)

Commit:
Can access 'Add Offer' as standard page without modal (#197)

@ruvenshilpakar, we're going to merge this and will be available in the next release (1.3.9).

Thank you!

salvoscala committed:
Let users access with email. (#195)

salvoscala committed:
Small theme fixes.

salvoscala committed:
Access to conversations (#198)

* Improve permissions to view a conversation

* Check permission on the form-submissions page

Commit:
Patch Drupal to resolve “DATE_RFC7231 already defined in PHP 7”

salvoscala committed:
Update the Roomify Channel Connector module. (#207)

salvoscala reviewed (approved):
looks good :)
